frontend/src/views/Org/AuditLogsPage/components/LogsTable.tsx

```diff
@@ -41,12 +41,15 @@ export const LogsTable = ({
 }: Props) => {
   const { currentWorkspace } = useWorkspace();
 
+  const filterProjectId =
```

Comment:
Nit: Confusing condition; I would break it up somehow so it's more readable.

Feel free to dismiss or update.